index.js

```diff
@@ -1,72 +1,137 @@
+    const book = new Book(title, author);
+    Store.addBook(book);
+    UI.displayBooks();
+    titleInput.value = '';
+    authorInput.value = '';
   }
 
-  getFromLocalStorage() {
-    const storedBooks = localStorage.getItem('books');
-    if (storedBooks !== null) {
-      this.books = JSON.parse(storedBooks);
+  static removeBook(target) {
+    if (target.classList.contains('delete')) {
+      const bookElement = target.parentElement;
+      const title = bookElement.querySelector('span:first-child').textContent.split(': ')[1];
+      const author = bookElement.querySelector('span:nth-child(2)').textContent.split(': ')[1];
+      const book = new Book(title, author);
+      Store.removeBook(book);
+      UI.displayBooks();
     }
   }
 }
-const bookCollection = new BookCollection();
-bookCollection.resetForm();
\ No newline at end of file
+
+// Event listeners
+document.addEventListener('DOMContentLoaded', UI.displayBooks);
+
+const addBookForm = document.getElementById('addBookForm');
+addBookForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  UI.addBook();
+});
+
+const bookList = document.getElementById('book-list');
+bookList.addEventListener('click', (e) => {
+  UI.removeBook(e.target);
+});
+const list = document.getElementById('list');
+list.addEventListener('click', (e) => {
+  e.preventDefault();
+  const booklist = document.querySelector('#book-list');
+  const addBookForm = document.querySelector('#addBookForm');
+  const contact = document.querySelector('.contact');
+  const heading = document.querySelector('#heading1');
+  booklist.style.display = 'block';
+  addBookForm.style.display = 'none';
+  heading.style.display = 'block';
+  contact.style.display = 'none';
+});
+
+const add = document.getElementById('add');
+add.addEventListener('click', (e) => {
+  e.preventDefault();
+  const addBookForm = document.querySelector('#addBookForm');
+  const heading = document.querySelector('#heading1');
+  const contact = document.querySelector('.contact');
+  addBookForm.style.display = 'flex';
+  const booklist = document.querySelector('#book-list');
+  booklist.style.display = 'none';
+  contact.style.display = 'none';
+  heading.style.display = 'none';
+});
+
+const contact = document.getElementById('contact');
+contact.addEventListener('click', (e) => {
+  e.preventDefault();
+  const addBookForm = document.querySelector('#addBookForm');
+  const heading = document.querySelector('#heading1');
+  const contact = document.querySelector('.contact');
+  addBookForm.style.display = 'none';
+  const booklist = document.querySelector('#book-list');
+  contact.style.display = 'flex';
+  booklist.style.display = 'none';
+  heading.style.display = 'none';
+});
+
+const date = document.getElementById('date');
+date.innerHTML = new Date().toLocaleString('en-US', {
+  month: 'long',
+  day: 'numeric',
+  year: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+  second: 'numeric',
+  hour12: true,
+});
```

Comment:
- [OPTIONAL] While you have displayed even the seconds, I highly recommend you to make sure the seconds are being changed on the screen after each second means your need to update time after 1 second. You can use the SetIntenval(), the JavaScript built-in method which repeats a block of code at every given timing event. Read [this lesson](https://developer.mozilla.org/en-US/docs/Web/API/setInterval) for more info.